docs/tutorials/picorv32_ram.rst

```diff
@@ -28,12 +28,13 @@ Before we add the complexity of a RAM macro block, let's build the core design u
 
     import siliconcompiler
 
-    chip = siliconcompiler.Chip('picorv32')
-    chip.load_target('skywater130_demo')
-    chip.input('picorv32.v')
-    chip.clock('clk', period=10)
-    chip.set('option', 'remote', True)
-    chip.run()
+    chip = siliconcompiler.Chip('picorv32')     # create chip object
+    chip.load_target('skywater130_demo')        # load predefined target
+    chip.input('picorv32.v')                    # define list of source files
+    chip.clock('clk', period=10)                # define clock speed of design
+    chip.set('option', 'remote', True)          # run remote in the cloud
+    chip.run()                                  # run chip compilation
+    chip.summary()                              # print results summary
```

Comment:
I think we still need to add a little more to indicate that at the end of this you will have an HTML and PNG summary

Reply:
This could be in the text above....